src/istgt_lu.c

```diff
@@ -4217,11 +4226,13 @@ maintenance_io_worker(void *arg)
 				} else if (spec->ex_state == ISTGT_LUN_OPEN_PENDING) {
 						do_open = 1;
 				}
+#ifndef	REPLICATION
```

Comment:
'unlock' need to be out of this '#ifndef'

Reply:
ok